packages/turbo-ignore/utils.js

```diff
@@ -0,0 +1,69 @@
+const fs = require("fs");
+const path = require("path");
+const { findRootSync } = require("@manypkg/find-root");
+
+export function searchUp(pathName, cwd) {
+  const root = path.parse(cwd).root;
+
+  let found = false;
+
+  while (!found && cwd !== root) {
+    if (fs.existsSync(path.join(cwd, pathName))) {
+      found = true;
+      break;
+    }
+
+    cwd = path.dirname(cwd);
+  }
+
+  if (found) {
+    return cwd;
+  }
+
+  return null;
+}
+
+export function getScope() {
+  if (process.argv.length > 1 && process.argv[2] != null) {
```

Comment:
Not in love with a utility function going looking for its own arguments. The simplest change I'd ship is passing in `process.argv.slice()` as an `argv` variable, but I still don't love that.